web/src/locales/en.json

```diff
@@ -88,6 +88,7 @@
     "helpUs": "Help us identify the problem by taking a look at the <a href=\"{{link}}\" target=\"_blank\">runtime logs</a> or contact the data provider.",
     "noParserInfo": "We do not yet have any information about this area.<br/> Want to help fix this? You can contribute by <a href=\"{{link}}\" target=\"_blank\">adding area data</a>.",
     "aggregationDisabled": "Data is not available for the selected time period due to missing granular breakdown. Try a broader time period.",
+    "fullyDisabled": "The data we have available for this zone is insufficient and therefore not shown. If you are aware of data sources for this area, please <a href=\"{{link}}\" target=\"_blank\">let us know</a>.",
```

Comment:
Just a small nitpick here, it wound be nice to use single quotes in the links so we can avoid the escape chars. But I see it's the same in all the other ones I guess it's fine.

Reply:
Yeah, I just copy-pasted - but happy to make an issue for generally fixing this problem afterwards :)

Reply:
Sounds good, something to do next slack week!

Reply:
Captured in Linear here: https://linear.app/electricitymaps/issue/AVO-336/remove-escaped-quotes-from-translation-files